### PR TITLE
Update README with env setup; clean up project.json and knexfile

### PR DESCRIPTION
**Pull request by BenHall-7**
# Description

Details the requirements of a local environment for running local databases and testing. Also cleans some mismatches in dependencies/devDependencies and the knexfile env variable names.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [x] Locally with `yarn test`, between me and @Hansen-Nick 

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts
- [x] I have checked that no one else is on staging and deployed this feature branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,35 @@ To get the server running locally:
...
+  c) Create two local databases, one for local development, and one for testing. You can do this with the `createdb` application in the terminal. If the username chosen was `postgres` when installing, then do `createdb -U postgres tidyhive`. You will be prompted for the password before execution. Do the same using `tidyhive-test` for the testing db.
+  d) ALTERNATIVELY: you can set the PGUSER and PGPASSWORD environment variables to skip username and password prompting.
+2) Setting up the .env file
+  a) In order to use purecrypt, two env variables are required. If you don't have PGUSER or PGPASSWORD env variables globally set, then you will need to include them here to access the databases. Other env variables needed are included. For ALGO, see the purecrypt documentation for available options:
+
+```none
```
Comment (Hansen-Nick) [inline README.md:28]: What is this none doing here?

Comment (Adammonast) [reply README.md:28]: ^

Comment (BenHall-7) [reply README.md:32]: From the markdown linter when the code block doesn't specify a language:
> MD040/fenced-code-language: Fenced code blocks should have a language specified
So I chose "none". After researching it, the correct choice may be "text" instead, so I will fix that

Comment (Hansen-Nick) [reply README.md]: Perfect, thanks. It was a tiny nitpick but I wasn't sure the proper usage. I agree it looks like text is the answer

**Pull request merged**